### PR TITLE
Add GPU and NFD configuration

### PR DESCRIPTION
**Pull request by strangiato**

### CODE REVIEW

**Comment by strangiato** [comment]
I'm not sure if there is an issue with the pipeline or if something is broken in the dependencies-install-cli.adoc file, but nothing is rendering after the first section in the preview.

Let me know if you see something obvious I am missing here.

**Review by rsriniva — APPROVED** [review]

**Comment by rsriniva** [comment]
> I'm not sure if there is an issue with the pipeline or if something is broken in the dependencies-install-cli.adoc file, but nothing is rendering after the first section in the preview.
> 
> Let me know if you see something obvious I am missing here.

Seems fine to me. Do you have a screenshot of the pages you are having problems with? maybe this is a browser cache issue?

**Review by jramcast — APPROVED** [review]
LGTM. I have tested the installation of the GPU and NFD operators via the web console.  All good. 

One quick question: The workbench creation form does not display the GPU selector, but I assume that's normal because there are no GPUs available in my compute nodes, right?

File: modules/chapter1/pages/dependencies-install-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+      #  klog:
+      #    addDirHeader: false
+      #    alsologtostderr: false
+      #    logBacktraceAt:
+      #    logtostderr: true
+      #    skipHeaders: false
+      #    stderrthreshold: 2
+      #    v: 0
+      #    vmodule:
+      ##   NOTE: the following options are not dynamically run-time
```
[the remark this answers is not in the document]
Comment (jramcast) [reply modules/chapter1/pages/dependencies-install-cli.adoc:196]: No big deal, I was just wondering. You can keep those lines, they might serve as a reference, I guess.

**Comment by strangiato** [comment]
> LGTM. I have tested the installation of the GPU and NFD operators via the web console. All good.
> 
> One quick question: The workbench creation form does not display the GPU selector, but I assume that's normal because there are no GPUs available in my compute nodes, right?

Correct.

I added a quick comment about that.

**Pull request merged**